apps/01PigLatin.js

```diff
@@ -7,11 +7,92 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer: You want to set the sting to all lower case so that if a user uses CAPS then it doens't
+  // matter becasue it will revert to lower case and the program will not crash.
+  //
+  word  = word.toLowerCase();
 
+  // Concept Check
+  //
+  // What is the significance of setting vowelIndex to -1?
+  // This might help:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+  //
+  // Your Answer: setting a variable equal to -1 is often used so that it can return a value if nothing was
+  // found, and since the index start at 0, -1 is used.
+  //
```

Comment:
We set `var vowelIndex = -1` as the initial state because that means we haven't found a vowel. `indexOf()` returns -1 when it can't find something.

If we make it thru the if statements, and `vowelIndex === -1`, then we found nothing.

If we make it thru the if statements, and `vowelIndex > -1`, we found a vowel.